src/isomorphic/modern/class/__tests__/ReactFragment-test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails react-core
+ */
+'use strict';
+
+let React;
+let ReactDOM;
+let createRenderer;
+let ReactTestRenderer;
+let ReactNoop;
+let ReactNative;
+let UIManager;
+let createReactNativeComponentClass;
+let ReactDOMFeatureFlags = require('ReactDOMFeatureFlags');
+let element;
+
+describe('ReactFragment', () => {
+  beforeEach(() => {
+    jest.resetModules();
+
+    React = require('react');
+
+    element = (
+      <React.Fragment>
+        hello <span>world</span>
+      </React.Fragment>
+    );
+  });
+
+  it('should render via native renderer', () => {
```

Comment:
Just the noop renderer is fine :D This doesn't touch any renderer-specific code.

Reply:
Should have said so sooner! :'(

Reply:
I’m sorry :( Didn’t know you were struggling with this. If something’s not renderer specific, we usually just test with the noop renderer. Except for older tests, which use React DOM.

Reply:
Gotcha, good to know! :)